src/main/java/com/fasterxml/jackson/core/StreamReadConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.fasterxml.jackson.core;
+
+/**
+ * The configuration for streaming reads.
+ *
+ * @since 2.15
+ */
+public class StreamReadConfig {
```

Comment:
Probably should have mentioned earlier but since there are limits/constraints, should name the container to indicate that.

Reply:
could you suggest a name - you suggested StreamReadConfig and I used that name based on your suggestion

Reply:
Sorry about that.

Maybe `StreamReadLimits` or `StreamReadConstraints`?